[V2] Update contributing.md (#546)
SalesforceCommerceCloud/pwa-kit: CONTRIBUTING.md

This repository is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) that includes the code that powers the PWA Kit. The code is divided into the following packages:

> - `template-retail-react-app`: A set of sample code and tooling for PWA Kit projects called the Retail React App

Reviewer: ```suggestion
- `template-retail-react-app`: A sample app that can be used as a starting point for a full retail storefront.
```

> - `template-express-minimal`: A set of sample code for a minimal express app

Reviewer: ```suggestion
- `template-express-minimal`: A sample app that can be used as a starting point for projects using express.
```

> - `pwa-kit-create-app`: A tool that generates PWA Kit projects based on the Retail React App

Reviewer 2: ```suggestion
- `pwa-kit-create-app`: A tool that generates PWA Kit projects based on the Retail React App template
```

> - `template-typescript-minimal`: A set of sample code for a minimal React app that support Typescript 

Reviewer: ```suggestion
- `template-typescript-minimal`: A sample app that can be used as a starting point for projects using TypeScript.
```

Author: We should mention that it is still a React app

- `pwa-kit-react-sdk`: A set of components and utilities for PWA Kit projects

> - `pwa-kit-runtime`: A packages that contains runtime environments code for apps to run in Managed Runtime

Reviewer 2: ```suggestion
- `pwa-kit-runtime`: A package that contains runtime environments for applications to run on Managed Runtime
```

> - `pwa-kit-cli`: A set of commands to build PWA Kit projects

Reviewer: ```suggestion
- `pwa-kit-cli`: A set of commands to build and manage PWA Kit projects
```

> - `template-express-minimal`: A sample app that can be used as a starting point for express projects.

Reviewer 2: ```suggestion
- `template-express-minimal`: A project template that can be used as a starting point for a generic Express application (does not include React).
```

> - `template-express-minimal`: A project template that can be used as a starting point for a generic Express application (does not include React).

Reviewer 2: ```suggestion
- `template-express-minimal`: A project template that can be used as a starting point for a generic Express application (does not include React)
```

> - `internal-lib-build`: A package that contains internal tools for other libraries in the monorepo 

Reviewer: ```suggestion
- `internal-lib-build`: A package that contains internal tools used by other libraries in the monorepo 
```

> - `template-retail-react-app`: A React sample app that can be used as a starting point for a full retail storefront.

Reviewer 2: ```suggestion
- `template-retail-react-app`: A project template that can be used as a starting point for a full retail storefront.
```

> - `template-retail-react-app`: A project template that can be used as a starting point for a full retail storefront.

Reviewer 2: ```suggestion
- `template-retail-react-app`: A project template that can be used as a starting point for a full retail storefront
```

> - `pwa-kit-runtime`: A packages that contains runtime environments code for apps to run in Managed Runtime

Reviewer: Would make sense to alphabetize the list.

> - `template-typescript-minimal`: A sample app that supports TypeScript can be used as a starting point for React projects.

Reviewer 2: ```suggestion
- `template-typescript-minimal`: A project template with TypeScript support that can be used as a starting point for React projects
```
